CONTRIBUTING.md

```diff
@@ -1,56 +1,54 @@
 # Vite Contributing Guide
 
-Hi! We are really excited that you are interested in contributing to Vite. Before submitting your contribution, please make sure to take a moment and read through the following guide:
+Hi! We're really excited that you're interested in contributing to Vite! Before submitting your contribution, please read through the following guide.
 
 ## Repo Setup
 
 The Vite repo is a monorepo using pnpm workspaces. The package manager used to install and link dependencies must be [pnpm](https://pnpm.io/).
 
 To develop and test the core `vite` package:
 
-1. Run `pnpm i` in Vite's root folder
+1. Run `pnpm i` in Vite's root folder.
 
 2. Run `pnpm run build` in Vite's root folder.
 
 3. If you are developing Vite itself, you can go to `packages/vite` and run `pnpm run dev` to automatically rebuild Vite whenever you change its code.
 
 You can alternatively use [Vite.js Docker Dev](https://github.com/nystudio107/vitejs-docker-dev) for a containerized Docker setup for Vite.js development.
 
-> Vite uses pnpm v7. If you are working on multiple projects with different versions of pnpm, it's recommend to enable [Corepack](https://github.com/nodejs/corepack) by running `corepack enable`.
+> Vite uses pnpm v7. If you are working on multiple projects with different versions of pnpm, it's recommended to enable [Corepack](https://github.com/nodejs/corepack) by running `corepack enable`.
 
 ## Debugging
 
-If you want to use break point and explore code execution you can use the ["Run and debug"](https://code.visualstudio.com/docs/editor/debugging) feature from vscode.
+To use breakpoints and explore code execution, you can use the ["Run and Debug"](https://code.visualstudio.com/docs/editor/debugging) feature from VS Code.
 
 1. Add a `debugger` statement where you want to stop the code execution.
 
-2. Click on the "Run and Debug" icon in the activity bar of the editor.
+1. Click the "Run and Debug" icon in the activity bar of the editor.
 
-3. Click on the "JavaScript Debug Terminal" button.
```

Comment:
This step isn't necessary to use the `debugger` statement or to use VS Code's debugger.

Reply:
That doesn't work for me. I had to create a javascript debug terminal in order for vscode to stop on the debugger statement.

Reply:
Ah, you're right. I had multiple instances of VS Code open, and must've tested that on the wrong one.